File: WebApplication1/WebApplication1/DTO/Response/BookingResDto.cs
using System.ComponentModel.DataAnnotations;
using WebApplication1.Models;

namespace WebApplication1.DTO.Response
{
    public class BookingResDto
    {
        public int BookingId { get; set; }
        public ConcertResDto Concert { get; set; } = null!;
        public string UserName { get; set; } = null!;
        public string UserEmail { get; set; } = null!;

        public BookingStatus Status { get; set; }
        public DateTime BookingTime { get; set; }
    }
}

File: angular-app/src/app/api/models/booking-res-dto.ts
/* tslint:disable */
/* eslint-disable */
import { BookingStatus } from './booking-status';
import { ConcertResDto } from './concert-res-dto';
export interface BookingResDto {
  bookingId?: number;
  bookingTime?: string;
  concert?: ConcertResDto;
  status?: BookingStatus;
  userEmail?: null | string;
  userName?: null | string;
}
